File: app/src/main/java/com/originb/inkwisenote/modules/ocr/data/Page.java
package com.originb.inkwisenote.modules.ocr.data;

import java.util.ArrayList;

public class Page{
    public double height;
    public double width;
    public double angle;
    public int pageNumber;
    public ArrayList<Word> words;
    public ArrayList<Span> spans;
    public ArrayList<Line> lines;
}

File: app/src/main/java/com/originb/inkwisenote2/modules/ocr/data/Page.kt
package com.originb.inkwisenote2.modules.ocr.data

class Page {
    var height: Double = 0.0
    var width: Double = 0.0
    var angle: Double = 0.0
    var pageNumber: Int = 0
    var words: ArrayList<Word>? = null
    var spans: ArrayList<Span>? = null
    var lines: ArrayList<Line>? = null
}
